extension/src/popup/components/signTransaction/Operations/KeyVal/index.tsx

```diff
@@ -386,7 +391,16 @@ export const KeyValueInvokeHostFnArgs = ({ args }: { args: xdr.ScVal[] }) => (
     <div className="OperationParameters">
       {args.map((arg) => (
         <div className="Parameter" key={arg.toXDR().toString()}>
-          {scValByType(arg)}
+          {arg.switch() === xdr.ScValType.scvAddress() ? (
```

Comment:
It seemed more appropriate to handle this here instead of in `scValByType` so we can keep that generic enough to work outside of components.